docs/user-guide/how_to_use_resource_strategy_fit_plugin.md

```diff
@@ -0,0 +1,372 @@
+# Resource Strategy Fit Plugin User Guide
```

Comment:
Please also add the SRA policy and example in this doc, refer to the resource-strategy-fit design doc.

Reply:
I would suggest splitting the SRA policy user guide into a separate document. Keeping it within the current one makes it too lengthy and less structured.